test/framework/config.go

```diff
@@ -0,0 +1,325 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+
+	"github.com/pkg/errors"
+	"sigs.k8s.io/cluster-api/test/framework/exec"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	// DefaultManagementClusterName is the default name of the Kind cluster
+	// used by the the e2e framework.
+	DefaultManagementClusterName = "mgmt"
```

Comment:
Since this is a test framework, should the default name here include the word test?

Reply:
Hi @detiber,

This is the same value as the original default, so I just didn't change it in order to keep things consistent.